src/main/java/me/itzg/ceres/CeresApplication.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package me.itzg.ceres;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import reactor.core.scheduler.Schedulers;
+
+@SpringBootApplication
+public class CeresApplication {
+
+	public static void main(String[] args) {
+		Schedulers.enableMetrics();
```

Comment:
Not seen this before

Reply:
You're going to be in for many surprises then ;) This app is based on all the latest and greatest including [Spring Reactive](https://spring.io/reactive). 

This specifically is documented https://projectreactor.io/docs/core/release/reference/#_scheduler_metrics